src/handlers/InvitesBtn.ts

```
@@ -24,35 +28,43 @@ export class InvitesButton extends ButtonHandler<[typeof InvitesId]> {
 export class InvitesAction extends ButtonHandler<[typeof InvitesActionId]> {
     public async execute(): Promise<void> {
+        await this.defer({ ephemeral: false })
         const { userId, choice } = this.params
         const settings = await database.findChannel(this.event.channelId)
-        if (!settings) return
-        if (this.event.user.id !== settings.ownerId) {
-            await this.event.reply("Only the channel owner can handle this request.")
-            return
+        const guild = await database.findServer(this.event.guildId)
+        if (!settings) throw new ChannelNotFound()
+        if (choice == "approve") {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)